### PR TITLE
Add API functionality to add an external user id to an existing user

### PR DESCRIPTION
**Pull request by maximilianschweitzer**

### CODE REVIEW

File: src/plenigo/services/UserManagementService.php [hunk 1]
```diff
@@ -134,15 +139,43 @@ public static function changeEmail($customerId, $email) {
...
+     * @param string $externalCustomerId external customer id
+     *
+     * @return bool TRUE external customer id added
+     *
+     * @throws PlenigoException In case of communication errors or invalid parameters
+     */
+    public static function addExternalCustomerId($customerId, $externalCustomerId)
+    {
+        $map = array(
+            'externalCustomerId' => $externalCustomerId
```
Comment (robertholfeld) [inline src/plenigo/services/UserManagementService.php:154]: Is it externalCustomerId or externalUserId?

File: src/plenigo/services/UserManagementService.php [hunk 2]
```diff
@@ -27,40 +27,44 @@
  * @author   Sebastian Dieguez <s.dieguez@plenigo.com>
  * @link     https://www.plenigo.com
  */
-class UserManagementService extends Service {
+class UserManagementService extends Service
+{
```
Comment (robertholfeld) [inline src/plenigo/services/UserManagementService.php:31]: Formatting?

File: src/plenigo/services/UserManagementService.php (hunk 1 above)
Comment (maximilianschweitzer) [reply src/plenigo/services/UserManagementService.php:154]: External customer id.

File: src/plenigo/services/UserManagementService.php (hunk 2 above)
Comment (maximilianschweitzer) [reply src/plenigo/services/UserManagementService.php:31]: Standard

**Pull request merged**